### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Thoth1111**
### Desktop Version

In this branch, I built up the desktop version of the portfolio using media queries. I also added my learning partners as collaborators to include a transition and animation. The changes included:

- Adding a media query of min-width 768px
- Modifying the landing page and styles to mimic figma template 3
- Modifying the work section, About me and skills section to mimic figma template 3
- Modifying the form section with the inclusion of an additional input and styling it to mimic figma template 3
- Working with my collaborators @alicemirigo92 and @dilsherb to add an animation and transition.
- Further modification of the form section, cleaning the code and fixing linter errors.

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -9,6 +9,13 @@
...
+    <header>
+        <ul id="navbar" type="none">
+            <li><a class="nav-icons" href="#">Portfolio</a></li>
+            <li><a class="nav-icons" href="#">About</a></li>
+            <li><a class="nav-icons" href="#">Contact</a></li>
+        </ul>
+    </header>
     <section class="toolbar">
         <p id="logo-text">Welcome</p>
         <div class="buttons" id="menu">
```
Comment (Kamba56) [inline Index.html]: - [x] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

File: Index.html [hunk 2]
```diff
@@ -128,24 +174,31 @@ <h2>About me</h2>
...
-                <input type="image"  src="Linkedin icon.png">
-                <input type="image"  src="medium.png">
-                <input type="image"  src="github.png">
-                <input type="image"  src="angellist.png"><br>
-            </div>
-            <p id="email">alwynsimiyu@gmail.com</p>
         </div>
-    </div> 
+        <div id="form-container">
+            <form class="form-entries">
```
Comment (Kamba56) [inline Index.html]: - [x] Please style the form section of your page to match the figma template.
### Your design



### Figma template
f1

File: Index.html [hunk 3]
```diff
@@ -128,24 +174,31 @@ <h2>About me</h2>
             </ul>
            </div> 
         </div>
-
```
Comment (Kamba56) [inline Index.html:131]: - [x] Kindly edit the alignment of your list of technologies to match the figa template.
### Your design

### fIGMA TEMPLATE
f2

File: Index.html [hunk 4]
```diff
@@ -9,6 +9,13 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
 </head>
 <body>
+    <header>
```
Comment (Kamba56) [inline Index.html]: - [x] There is soo much blank space at the top of your header, kindly remove that please to match the figma template. You are expected to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
### Your design

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @Thoth1111 , 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great use of Github workflow!👏👏👏
- [x] All linters passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 4 above)
Comment (Thoth1111) [reply Index.html]: Hello @Kamba56, 
Thanks for the review. The space at the top of the mobile version is actually part of the figma template. In it were a battery icon, time, and network signal indicators which I was told I should not include in previous reviews. However, I will go ahead and remove the space.

Comment (Kamba56) [reply Index.html]: Great. Those icons were only to show a great representation of the mobile phone in the figma template. You don't need it in your design 😁

File: Index.html (hunk 1 above)
Comment (Thoth1111) [reply Index.html]: Thank you. I have fixed the indentation now.

File: Index.html (hunk 2 above)
Comment (Thoth1111) [reply Index.html]: Thank you. I have removed the flex-wrap property that was responsible for the issue. The form now closely mimics the template in positioning

File: Index.html (hunk 3 above)
Comment (Thoth1111) [reply Index.html:131]: Thank you. I have added justified the content of the list to center which has aligned them correctly.

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @Thoth1111    👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️ Professional Pull Request 👍
✔️ No Linter errors 👍   

## Optional Requirements 

- [ ] [OPTIONAL] I think it would be better to provide a valid **LICENSE** link in your **README** as open source software need to be licensed to provide rights to **modify**, **distribute** other than creator :joy: 
 
image

 _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**